siac/rentercmd.go

```diff
@@ -245,18 +245,18 @@ func rentersetallowancecmd(amount, period string) {
 	fmt.Println("Allowance updated.")
 }
 
-// byHeight sorts contracts by their expiration, high to low. If two contracts
-// expire at the same height, they are sorted by their host's address.
-type byHeight []api.RenterContract
-
-func (s byHeight) Len() int      { return len(s) }
-func (s byHeight) Swap(i, j int) { s[i], s[j] = s[j], s[i] }
-func (s byHeight) Less(i, j int) bool {
-	hi, hj := s[i].EndHeight, s[j].EndHeight
-	if hi == hj {
+// byValue sorts contracts by their value in siacoins, high to low. If two
+// contracts have the same value, they are sorted by their host's address.
+type byValue []api.RenterContract
```

Comment:
agreed, but that seems to be the standard for sorting types. And it helps that the type is defined close to where it's used.